python-packages/fle_utils/chronograph/management/commands/cronserver_blocking.py

```diff
@@ -37,7 +38,8 @@ def handle(self, *args, **options):
                 logging.debug("No jobs due to run.")
             
             # Sleep a little bit at a time to discover if we have to shutdown
-            for __ in range(6):
-                time.sleep(sleep_time // 6)
+            for __ in range(60):
```

Comment:
It's supposed to sleep very little otherwise it won't see the shutdown flag. I mistook the default setting to be 60, it was actually set to 600 in our complex setting stack :)